client/src/constants/action_types.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
I came across [tcomb](https://github.com/gcanti/tcomb/blob/master/GUIDE.md#the-enums-combinator) this weekend, and it seems like it has some cool features. One of them being Enums, and pattern matching. Don't know if you want to use it, but it may be worth it.